components/helptext/src/styles/tokens.scss

```diff
@@ -0,0 +1,9 @@
+@import "@aurodesignsystem/design-tokens/dist/tokens/SCSSVariables";
+
+:host {
+  --ds-auro-help-text-color: var(--ds-color-text-tertiary-default, #{$ds-color-text-tertiary-default});
```

Comment:
Make this `--ds-auro-helptext-color` since we are calling the component `auro-helptext`